src/main/java/lotto/domain/LottoGames.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class LottoGames implements Iterable<LottoGame>{
+    private List<LottoGame> lottoGames = new ArrayList<>();
+
+    public LottoGames(int money) {
+        for (int currentMoney = money; currentMoney >= LottoGame.PRICE; currentMoney -= LottoGame.PRICE) {
+            lottoGames.add(new LottoGame());
+        }
+    }
```

Comment:
for 문의 조건들이 가독성이 조금 떨어지는 것 같아요!
다음과 같은 방식은 어떠신가요? 😄 
```suggestion
    public LottoGames(int money) {
        int lottoCount = money/LottoGame.PRICE;

        for (int i = 0; i < lottoCount ; i++) {
            lottoGames.add(new LottoGame());
        }
    }
```

Reply:
훨씬 좋은 방식입니다 ㅎㅎ